plugins/org.jboss.tools.livereload.core/META-INF/MANIFEST.MF

```diff
@@ -36,7 +36,10 @@ Require-Bundle: org.eclipse.ui,
  org.eclipse.jst.server.tomcat.core;bundle-version="1.1.500",
  com.fasterxml.jackson.core.jackson-annotations;bundle-version="2.5.0",
  com.fasterxml.jackson.core.jackson-core;bundle-version="2.5.0",
- com.fasterxml.jackson.core.jackson-databind;bundle-version="2.5.0"
+ com.fasterxml.jackson.core.jackson-databind;bundle-version="2.5.0",
+ org.apache.aries.spifly.dynamic.bundle;bundle-version="1.0.2",
+ org.apache.aries.util;bundle-version="1.1.1",
+ org.objectweb.asm.all;bundle-version="5.0.4"
```

Comment:
If I dont' explicitly add those 2 dependencies (aries + its 2 required bundles), running the tests will fail because no osgi.serviceloader.registrar will be available.
If the tests are run with all the TP plugins in a runtime Eclipse, this problem will not be seen. In my case, I enforce the requirements.